app/models/manageiq/providers/vmware/cloud_manager/vm/operations.rb

```diff
@@ -1,3 +1,15 @@
 module ManageIQ::Providers::Vmware::CloudManager::Vm::Operations
+  extend ActiveSupport::Concern
+
   include_concern 'Power'
+
+  def raw_destroy
+    raise "VM has no #{ui_lookup(:table => "ext_management_systems")}, unable to destroy VM" unless ext_management_system
+    with_provider_object(&:undeploy)
+
```

Comment:
Nitpicking, but could you please remove this empty line?

Reply:
Empty line removed